addon/templates/components/flexberry-ddau-slider.hbs

```diff
@@ -0,0 +1,15 @@
+{{!--
+  Binding through get-helper value=(get this "value") is a way to achieve one way binding.
+  It makes value immutable within component, what is major rule of "Data Down Actions Up (DDAU)" pattern:
+  component should display data, but not mutate it by its own.
+--}}
+{{ui-slider
```

Comment:
It is possibly good idea and the way to make all the controls with the same framework, but there were so much problems with semantic's elements inside accordion and I believe it doesn't worth time to try